docs/opentelemetry.md

```diff
@@ -0,0 +1,68 @@
+# OpenTelemetry
+
+[OpenTelemetry](https://opentelemetry.io/) is a set of libraries, agents, and instrumentation to provide observability (metrics, logs and traces) in applications.
+
+`containerd-shim-wasm` crate has a set of APIs to enable OpenTelemetry tracing in the shim. This document is a guide on how to use OpenTelemetry tracing in your shim.
+
+## Usage
+
+To use OpenTelemetry tracing in your shim, you need to use the `opentelemetry` feature in the `containerd-shim-wasm` crate and the `tokio` crate.
+```toml
+containerd-shim-wasm = { workspace = true, features = ["opentelemetry"] }
```

Comment:
will the `tracing` feature and `opentemetry` feature always be turned on together? should they be the same?

Reply:
yeah I need to update this doc it's not update to date with the work that turns tracing to an optional feature.

Reply:
is the plan to merge the features or keep them seperate?

Reply:
I am inclined to merge them, but this may change once we use `tracing::log` to replace `log` crate